### PR TITLE
Bump eslint-plugin-vue

### PR DESCRIPTION
**Pull request by larkinscott**
Bump eslint-plugin-vue to 4.2.2.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "eslint-plugin-sorting": "^0.3.0",
     "eslint-plugin-standard": "^3.0.0",
     "eslint-plugin-unicorn": "^2.1.2",
-    "eslint-plugin-vue": "^2.0.1",
+    "eslint-plugin-vue": "^4.2.0",
```
Comment (chrishulton) [inline package.json]: my only concern with such a big version bump is if there are breaking changes that would impact projects using the old version.  worth skimming the plugin changelog to see if anything jumps out.

if that seems ok, LGTM.

**Comment by larkinscott** [comment]
Skimmed the changelog for breaking changes and tested locally. All seems well.

**Comment by filipalacerda** [comment]
Thank you @larkinscott :heart:

**Comment by dzaporozhets** [comment]
Thanks!

**Pull request merged**